pages/documentation/developer/repositories/powsybl-single-line-diagram_guide1.md

```diff
@@ -0,0 +1,114 @@
+# Generating my first single line diagrams - Guide 1
+
+In this guide we create a few single line diagrams of a test network with [powsybl-single-line-diagram](powsybl-single-line-diagram.md).
+
+
+First of all, we need to add a Maven dependency for the Network model and three additional ones for this example: two for the `Network` test case, one for simple logging capabilities:
+
+```xml
+<dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-iidm-impl</artifactId>
+    <version>3.7.1</version>
+</dependency>
+<dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-iidm-test</artifactId>
+    <version>3.7.1</version>
+</dependency>
+<dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-config-test</artifactId>
+    <version>3.7.1</version>
+</dependency>
+<dependency>
+    <groupId>org.slf4j</groupId>
+    <artifactId>slf4j-simple</artifactId>
+    <version>1.7.22</version>
+</dependency>
+```
+
+We can now load the node/breaker test `Network` we are interested in:
+```java
+Network network = FictitiousSwitchFactory.create();
+```
+
+In order to be able to generate the diagrams, we need to add the single line diagram core module dependency:
+```xml
+<dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-single-line-diagram-core</artifactId>
+    <version>1.7.2</version>
+</dependency>
+```
```

Comment:
Maybe we should move this in the previous section: add all the required dependencies once.

Reply:
I merged all the dependencies of two guides into one section